src/test/bin/winkernel/control.cpp

```diff
@@ -1259,6 +1259,10 @@ QuicTestCtlEvtIoDeviceControl(
         QuicTestCtlRun(QuicTestVersionStorage());
         break;
 
+    case IOCTL_QUIC_RUN_CONNECT_AND_IDLE_FOR_DEST_CID_CHANGE:
```

Comment:
You're missing the `0,` above in the size array now, right? Did you delete that?

Reply:
No, I merged the main branch into this branch, there was a change in this file, so I forgot to add it. I'll add it now